### DISCUSSION TITLE
Welcome to TQUIC Discussions!

### DISCUSSION THREAD

**Discussion by iyangsj** [Tencent/tquic#14] [Announcements（公告）]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

**Comment by mingjiana** [comment]
这个协议其他语言的项目可以用吗？比如golang语言的项目，如果可以的话要怎么用呢？如果想把转认的传输层协议换成这个可以不？

**Reply by iyangsj** [reply]
请创建单独的帖子 https://github.com/Tencent/tquic/discussions/new/choose 讨论 
TQUIC目前不支持golang，golang语言项目可以考虑官方的[net-x-go](https://pkg.go.dev/golang.org/x/net)或[quic-go](https://pkg.go.dev/github.com/lucas-clemente/quic-go)
